web/components/Landing/ContactForm.js

```diff
@@ -1,22 +1,69 @@
+import classnames from 'classnames';
 import Logo from '../../images/logo.svg';
+import MailchimpSubscribe from 'react-mailchimp-subscribe';
+
+import { Formik } from 'formik';
+import { string, object } from 'yup';
+
+const validationSchema = object().shape({
+  email: string().email().required(),
+});
 
 export default function ContactForm() {
   return (
-    <form className="w-1/4 py-10 mx-auto">
-      <h2 className="font-museo w-content flex mx-auto text-5xl font-bold text-blue-500">
-        <Logo className="mr-10" />
-        Join us now
-      </h2>
+    <MailchimpSubscribe
+      url={process.env.NEXT_PUBLIC_MAILCHIMP_URL}
+      render={({ subscribe, status, message }) => (
+        <Formik
+          initialValues={{ email: '' }}
+          onSubmit={({ email }) => subscribe({ EMAIL: email })}
+          validationSchema={validationSchema}
+        >
+          {({
+            handleSubmit,
+            values,
+            errors,
+            touched,
+            handleChange,
+            handleBlur,
+          }) => (
+            <form className="w-1/4 py-10 mx-auto" onSubmit={handleSubmit}>
+              <h2 className="font-museo w-content flex mx-auto text-5xl font-bold text-blue-500">
+                <Logo className="mr-10" />
+                Join us now
+              </h2>
 
-      <div className="flex flex-col items-center justify-center mt-10 space-y-10">
-        <input
-          type="text"
-          className="hover:bg-blue-100 w-full border-0 border-b border-blue-500 border-solid"
-        />
-        <button className="px-5 py-2 text-white bg-yellow-500 rounded-sm">
-          Join the mailing list
-        </button>
-      </div>
-    </form>
+              <div className="flex flex-col items-center justify-center mt-10 space-y-10">
+                <input
```

Comment:
for the first time, I couldn't figure that this is an input field only when I clicked on the "join the mailing list", it looks like a divider, not an input field

Reply:
https://github.com/tezos-israel/wantez/issues/19